src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -556,6 +564,33 @@ public GHLicense getLicense(String key) throws IOException {
         return r;
     }
 
+    /**
+     * Alias for {@link #getUserPublicOrganizations(String)}.
+     */
+    public Map<String, GHOrganization> getUserPublicOrganizations(GHUser user) throws IOException {
+        return getUserPublicOrganizations( user.getLogin() );
+    }
+
+    /**
+     * This method returns a shallowly populated organizations.
+     *
+     * To retrieve full organization details, you need to call {@link #getOrganization(String)}
+     * TODO: make this automatic.
+     *
+     * @param user the user to retrieve public Organization membership information for
+     *
+     * @return the public Organization memberships for the user
+     */
+    public Map<String, GHOrganization> getUserPublicOrganizations(String login) throws IOException {
```

Comment:
The other method takes a plain old `String`, so that users' code doesn't have to construct a `GHUser` just to look up organizations.